packages/api-client/src/views/Request/libs/request.ts

```diff
@@ -0,0 +1,21 @@
+import type { RequestExampleParameter } from '@scalar/oas-utils/entities/spec'
+
+/**
+ * Check if a RequestExampleParameter has any of the following properties:
+ * - description
+ * - type
+ * - default
+ * - format
+ * - minimum
+ * - maximum
+ */
+export const hasItemProperties = (item: RequestExampleParameter): boolean => {
+  return !!(
+    item.description ||
+    item.type ||
+    item.default ||
+    item.format ||
+    item.minimum ||
+    item.maximum
+  )
+}
```

Comment:
```suggestion
export const hasItemProperties = (item: RequestExampleParameter) => 
  Boolean(
    item.description ||
    item.type ||
    item.default ||
    item.format ||
    item.minimum ||
    item.maximum
  )
```
I would write it like this personally but functionally is the same

Reply:
I agree it's more clear too